src/commands/blocks-isolated.ts

```
@@ -0,0 +1,457 @@
+			mode: ctx.values.mode,
+			order: ctx.values.order,
+			offline: ctx.values.offline,
+			sessionDurationHours: ctx.values.sessionLength,
+		});
+
+		if (blocks.length === 0) {
+			if (ctx.values.json) {
+				log(JSON.stringify({ blocks: [] }));
+			}
+			else {
+				logger.warn('No Claude usage data found in isolated directories.');
```

ESLint

[error] Unsafe call of a(n) `error` type typed value.

(ts/no-unsafe-call)